submissions/dashakim/tiny-world/index.js

```diff
@@ -0,0 +1,64 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+    constructor(species, name, gender, legs, hands, saying) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.legs = legs;
+        this.hands = hands;
+        this.saying = saying
+    }
+}
+
+const man = new Inhabitant('human', 'Leonid', 'male', '2', '2', 'Hello everyone!')
+const woman = new Inhabitant('human', 'Daria', 'female', '2', '2', 'Good day!')
+const dog = new Inhabitant('dog', 'Archi', 'male', '4', '0', 'Woof-woof')
+const cat = new Inhabitant('cat', 'Co-co', 'female', '4', '0', 'Meow')
+const catWoman = new Inhabitant('cat-woman', 'Kitty', 'female', '2', '2')
+man.friends = [woman.name, dog.name].join(', ')
+woman.friends = [cat.name]
+cat.friends = [man.name, woman.name].join(', ')
+dog.friends = [man.name, woman.name, cat.name].join(', ')
```

Comment:
This construction keeps friends' names rather than friends per se.
So if any friend changes her/his name the list won't reflect the changes.
Keeping a list of friends will keep things in sync. Of course you will require a class method to convert a list of friends into a comma-delimited string with their names.

BTW, consider the following snippet:
`[obj1, obj2, obj3].map(obj => obj.name).join(', ');`

Reply:
I decided to delete friendship for current task.  I need to re-write all task where friendship would look like more logic